README.md

```diff
@@ -27,13 +27,14 @@ This is the GitHub repository of Julia source code, including instructions for c
 - **Packages:** <http://pkg.julialang.org/>
 - **Source code:** <https://github.com/JuliaLang/julia>
 - **Git clone URL:** <git://github.com/JuliaLang/julia.git>
+- **Discussion forum:** <https://discourse.julialang.org>
 - **Mailing lists:** <http://julialang.org/community/>
 - **Gitter:** <https://gitter.im/JuliaLang/julia>
 - **IRC:** <http://webchat.freenode.net/?channels=julia>
 - **Code coverage:** <https://coveralls.io/r/JuliaLang/julia>
 
-The mailing list for developer discussion is
-<http://groups.google.com/group/julia-dev/>. All are welcome, but the volume
+The forum for developer discussion is
+<https://discourse.julialang.org/c/dev/>. All are welcome, but the volume
 of messages is higher, and the discussions tend to be more esoteric. New
```

Comment:
I assume the text was originally contrasting with julia-users, so I guess in this case it would be referring to a higher volume than the equivalent Discourse category?